doc/topics/index.rst

```diff
@@ -110,6 +110,12 @@ is hosted by Google Groups. It is open to new members.
 
 .. _`salt-users mailing list`: https://groups.google.com/forum/#!forum/salt-users
 
+Additionally, all users of Salt should be subscribed to the Announcements mailing
+list which contains important updates about Salt, such as new releaes and
+security-related announcements. This list is low-traffic.
+
+.. _`salt-announce mailing list`: https://groups.google.com/forum/#!forum/salt-announce
+
```

Comment:
Very good idea! As we're looking at this, might be worthy adding a link to the (still very obscure) repository https://github.com/saltstack/salt-enhancement-proposals/ where critical decisions happen, yet it's still followed only by 30 people - majoritarily SaltStack employees (I've only found out about its existence by mistake, after several important SEPs were discussed).

Reply:
Great idea! I've pushed a change to include this infomration in a block of text just below this one.